### ISSUE TITLE
Common_Engine: Area() returns wrong results for self-intersecting outlines

### ISSUE THREAD

**Issue by pawelbaran** [BHoM/BHoM_Engine#723]
### This issue has been archived 

Historic issue archive can be found [here](https://burohappold.sharepoint.com/sites/BHoM/01_Master%20File/Forms/AllItems.aspx?id=%2Fsites%2FBHoM%2F01_Master%20File%2F999_BHoM_GitHub_History_Archive)

Same to the `Centroid` etc. To be resolved once `IsSelfIntersecting` is in place, see #856

**Comment by LMarkowski** [comment]
No longer relevant. Methods moved from `Common` to `Spatial` and now return correct results or proper warnings.
